movies/src/components/Movie.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default (props) => {
+    const state = props.location.state.movie;
+    const image = {
+        backgroundImage: 'url(' + state.img + ')',
+    }
+    return (
+        <div className="this">
```

Comment:
Generally, avoid extremely generic class names like `this`, as they are entirely not descriptive of this HTML element.